components/eventing-controller/controllers/subscriptionv2/jetstream/test/reconciler_integration_test.go

```diff
@@ -514,88 +516,86 @@ func TestChangeSubscription(t *testing.T) {
 
 	for _, tc := range testCases {
 		t.Run(tc.name, func(t *testing.T) {
-			// The new t instance needs to be used for making assertions. If ignored, errors like this will be printed:
-			// testing.go:1336: test executed panic(nil) or runtime.Goexit: subtest may have called FailNow on a parent test
-			ens := ens
-			ens.G = gomega.NewGomegaWithT(t)
+			t.Parallel()
+			g := gomega.NewGomegaWithT(t)
 
 			// given
```

Comment:
these given / when / then comments seem off
given: setup of the test
when: lets do something
then: assert that the correct thing happens.

in this case here we have multiple assertions already in the given block

Reply:
updated them.